### DISCUSSION TITLE
LF7 Feedback

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#146] [Show and tell]
https://teambasedinquirylearning.github.io/precalculus/2024/LF7.html

Author: @kathypinzon  
Reviewer: @tdegeorge

**Comment by tdegeorge** [comment]
Activity 3.7.2: Should we provide an answer to part (c)? The child admission price is $24.50 and the adult admission price is $15.25. 

So...not sure we care about this, but maybe switch the numbers around so that the adult price is more expensive? Doesn't seem right to have the child's admission price to cost more :-p

**Reply by kathypinzon** [reply]
I wanted kids more expensive because at a fair they ride more stuff......

**Comment by tdegeorge** [comment]
Did you mean to include Activity 1.2.9 in the text? Just wondering because you didn't do that for the previous problem (Activity 3.6.2), but when I click on it, it appears within the text in the instructor version.

**Reply by kathypinzon** [reply]
I have no idea... I just referenced it like I did the other so not sure why it shows up like that.  @siwelwerd any thoughts?

**Reply by siwelwerd** [reply]
I'm not sure what we mean here -- possible the numbers have changed, but the reference I see to Activity 1.2.7 (in Activity 3.7.3) is collapsed.  You can expand it by clicking on it if you want.  Is this not an okay behavior?

**Comment by tdegeorge** [comment]
Activity 3.7.4: The $104,000 and $16,000 need to be in math mode. Same with the answer choices in part (b) and the answer for part (b).

**Comment by tdegeorge** [comment]
Activity 3.7.5: Need the math mode for the numbers. Company A should be capitalized.

The answer should be 1,000 suits (not 100). Should we also include the system set up in the answer?
